docs/generated/packages/nx-plugin/executors/e2e.json

```diff
@@ -11,20 +11,23 @@
     "properties": {
       "target": {
         "description": "The build target for the Nx Plugin project.",
-        "type": "string"
+        "type": "string",
+        "x-priority": "important"
       },
       "jestConfig": {
         "type": "string",
         "description": "Jest config file.",
         "x-completion-type": "file",
-        "x-completion-glob": "jest.config@(.js|.ts)"
+        "x-completion-glob": "jest.config@(.js|.ts)",
+        "x-priority": "important"
       },
       "tsSpecConfig": {
         "type": "string",
         "description": "The tsconfig file for specs.",
         "x-deprecated": "Use the `tsconfig` property for `ts-jest` in the e2e project `jest.config.js` file. It will be removed in the next major release.",
```

Comment:
Why is this marked as both `deprecated` and `important`? Seems counter-intuitive to me